### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by peterchappy**
I've mainly been just chipping away at this a little bit every so often. Trying to put a lot of love into it and just figuring stuff out as I go. I'm gonna comment some random questions I have and maybe you can help. I haven't done much of any browser QA and responsiveness, but I am keeping that in mind.

Also, I'm not sure your WebPack setup, but I'm just uses a basic project generator so after you install, all you have to do is `npm start`

## Some known issues

- Media query issues w/ `Navigation`
- `Navigation` not properly position absolute (had it working, but I messed it up at some point refactoring)
- Text not centered in the `TypedText`
- `TypedText` not rendering correctly when you shrink the browser. 

## Questions I have

- In `ContentBlock.css` why is the following not being applied?
```
.contentBlockFlexContainer:first-child{
    margin-bottom: 10%;
}
```

- Do you have any idea about the `TypedText` not rendering correctly?
- I'm curious to get you opinion on `type()` in `TypedText.js`

### CODE REVIEW

File: src/components/contentblocks/SocialIcon.js [hunk 1]
```diff
@@ -3,7 +3,6 @@ import PropTypes from 'prop-types';
 import './SocialIcon.css';
 
 function SocialIcon({ icon }) {
-
   const { link, img, type } = icon;
```
Comment (Gsarault44) [inline src/components/contentblocks/SocialIcon.js]: Not sure why you need this here, if you change it over to just {...props} you shouldn;t need this.

**Comment by peterchappy** [comment]
So I was using the 
```
const NavigationItem = ({ title }) => {
}
```

and it seemed that the Linter was yelling at me and when I changed it worked for me. Maybe I screwed up my linter? I'll take another look tonight.

File: src/components/contentblocks/SocialIcon.js (hunk 1 above)
Comment (peterchappy) [inline src/components/contentblocks/SocialIcon.js]: Not sure what your trying to explain here?  I'm using de-referencing to pull out the `icon` object from `props`, and then again with `const { link, img, type } = icon;` to keep my code clean so I don't have to do `props.icon.link whenever I want to access one of these variables`

Comment (Gsarault44) [inline src/components/contentblocks/SocialIcon.js]: I guess its fine either way, this is what i meant 
```
function SocialIcon({ ...props }) {
  return (
    <div className="socialIcon">
      <a href={props.link}>
        <img src={props.img} alt={props.type} />
      </a>
    </div>
  );
}

SocialIcon.propTypes = {
    link: PropTypes.string.isRequired,
    img: PropTypes.string.isRequired,
    type: PropTypes.string.isRequired
};


```

**Comment by Gsarault44** [comment]
your linter might be right, cuz Im pretty sure we are using the same linter, cuz we got ours from airbnb too.

**Comment by peterchappy** [comment]
Is it giving you errors for fat arrow functions then?

File: src/components/contentblocks/SocialIcon.js (hunk 1 above)
Comment (peterchappy) [inline src/components/contentblocks/SocialIcon.js]: Ahhh I see aka get rid of the parent icon

**Comment by peterchappy** [comment]
Wish we were both learning this stuff while you were still at BC, would have been fun

**Pull request merged**